4337/contracts/test/TestSingletonSigner.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+/* solhint-disable one-contract-per-file */
+pragma solidity >=0.8.0;
+
+import {ISignatureValidator} from "@safe-global/safe-contracts/contracts/interfaces/ISignatureValidator.sol";
+
+contract TestSingletonSigner is ISignatureValidator {
+    struct Key {
+        uint256 _dummy;
```

Comment:
This is just to verify this model would work for P256 signatures (which require 2 `uint256` slots, one for the X and one for the Y coordinate of the public key - which is a point on the curve).